### ISSUE TITLE
Metro Exodus on GTX 760 2GB

### ISSUE THREAD

**Issue by MrArtemSid** [doitsujin/dxvk#2493]
Metro Exodus crashes on DXVK, but works fine using VKD3D on ASUS GTX 760

### Software information
Game: Metro Exodus 

### System information
- GPU: ASUS GTX 760 2 GB
- Driver: 470
- Wine version: any
- DXVK version: latest
- System: Manjaro KDE / Windows 10 

### Log files
- d3d9.log: 
- d3d11.log: [MetroExodus_d3d11.log](https://github.com/doitsujin/dxvk/files/8054335/MetroExodus_d3d11.log)
- dxgi.log: [MetroExodus_dxgi.log](https://github.com/doitsujin/dxvk/files/8054336/MetroExodus_dxgi.log)

**Comment by MrArtemSid** [comment]
> > err:   DxvkSubmissionQueue: Command submission failed: VK_ERROR_DEVICE_LOST
> 
> Driver crashes for some reason, not really something we can fix since it works on other GPUs.
> 
> We also cannot support GPUs that no longer have active driver support from Nvidia.

Is it possible to somehow debug this issue? Or nvidia proprietary drivers aren't possible to debug?

**Comment by MrArtemSid** [comment]
> > err:   DxvkSubmissionQueue: Command submission failed: VK_ERROR_DEVICE_LOST
> 
> Driver crashes for some reason, not really something we can fix since it works on other GPUs.
> 
> We also cannot support GPUs that no longer have active driver support from Nvidia.

Somehow got dxvk working by using portproton https://github.com/Castro-Fidel/PortWINE

[MetroExodus_dxgi.log (1).txt](https://github.com/doitsujin/dxvk/files/8055313/MetroExodus_dxgi.log.1.txt)
[MetroExodus_d3d11.log (1).txt](https://github.com/doitsujin/dxvk/files/8055314/MetroExodus_d3d11.log.1.txt)

**Comment by doitsujin** [comment]
> Is it possible to somehow debug this issue? Or nvidia proprietary drivers aren't possible to debug?

Not really, DXVK is known to be unstable on Nvidia cards with small VRAM though so we typically recommend to not run it on cards with less than 4GB.

**Comment by MrArtemSid** [comment]
Okay

**Comment by aabbtree77** [comment]
I also use GTX 760 and have experienced similar crashes with the newest dxvk v2.3. 

Downgrading to dxvk v1.10.2 solves all my problems and improves the default wine d3d11 rendering substantially.

Ubuntu 22.04, wine-8.0.2. See the comment by [Blisto91](https://github.com/Blisto91), issue #2727.
